tox.ini

```diff
@@ -1,11 +1,5 @@
 [tox]
-envlist = py{27,36,37,38}-sphinx{16,17,18,20,21,22,23,24,30,31,32}
-
-[tox:travis]
-2.7 = py27-sphinx{16,17,18,20,21,22,23,24,30,31,32}
-3.6 = py36-sphinx{16,17,18,20,21,22,23,24,30,31,32}
-3.7 = py37-sphinx{16,17,18,20,21,22,23,24,30,31,32}
-3.8 = py38-sphinx{16,17,18,20,21,22,23,24,30,31,32}
+envlist = py{27,36,37,38,39}-sphinx{16,17,18,20,21,22,23,24,30,31,32}
```

Comment:
I realize this isn't really a change that you made but I'm surprised `py27-sphinx-3*` work at all since I think Sphinx3 is Python3 only.